src/processor.rs

```diff
@@ -1,69 +1,53 @@
-use tokio::io::AsyncWriteExt;
-use tokio::net::TcpStream;
+use anyhow::Result;
```

Comment:
How is `anyhow::Result` different from normal `Result`?

Reply:
It captures any of the error types be that: python errors or unwrap errors etc. so it means I can capture them all and we can capture them to display a nice error rather than crashing